src/gazebo/ros_gz_launch/launch/competition_field.launch.py

```diff
@@ -94,9 +94,9 @@ def generate_launch_description():
 
     return LaunchDescription([
         gz_sim,
-        # DeclareLaunchArgument('rviz', default_value='true', description='Open RViz.'),
+        DeclareLaunchArgument('rviz', default_value='true', description='Open RViz.'),
```

Comment:
Why do we have this and line 102 with rviz being started

Reply:
ros2 launch ros_gz_launch competition_field.launch.py rviz:=false will not launch rviz

Reply:
Can you move this into the rviz declaration then so its not here, it's a bit confusing

Reply:
Done!